src/signals/incident/containers/IncidentContainer/services.js

```diff
@@ -1,7 +1,7 @@
 const fieldTypeMap = {
   checkbox_input: 'CheckboxInput',
   date_time_input: 'DateTimeInput',
-  description_with_classification_input: 'DescriptionWithClassificationInput',
+  description_with_classification_input: 'DescriptionInput',
```

Comment:
Shouldn't this be 'DescriptionInputRenderer' instead of 'DescriptionInput'?

Reply:
Indeed, not sure whether this part is used anywehre...